syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.zb_stats";
option java_outer_classname = "ZBStatsProto";
option objc_class_prefix = "ZBS";

package zb_stats;

service StatsReport {
  rpc GetStats (StatsRequest) returns (StatsReply) {}

}

message StatsRequest {
  string type = 1;
}

message StatsItem {
    int64 timestamp = 1;
    int32 istat1 = 2;
    int32 istat2 = 3;
    float fstat1 = 4;
    float fstat2 = 5;
}
message StatsReply {
    map<string, StatsItem> stats = 1;
}
